code/LeftAndMain.php

```diff
@@ -349,11 +349,10 @@ protected function init()
 
             // Bootstrap components
             Requirements::javascript('silverstripe/admin: thirdparty/popper/popper.min.js');
-            Requirements::javascript('silverstripe/admin: thirdparty/bootstrap/js/dist/util.js');
```

Comment:
As per https://github.com/orgs/twbs/discussions/33152 the util.js file is no longer needed